hypervisor/arch/x86/guest/vm.c

```diff
@@ -368,6 +368,9 @@ void vm_fixup(struct vm *vm)
 		int i;
 
 		vm_desc = get_vm_desc(0);
+		if (vm_desc == NULL)
+			return;
```

Comment:
in this case, 'vm_desc' will never be NULL, as the index is 0,  it is just to clear Klocwork warning.  
anyway, will add pr_err , in case that 'get_vm_desc( )' may be revised. @dongyaozu